scikeras/wrappers.py

```diff
@@ -124,8 +125,12 @@ def __init__(self, build_fn=None, **sk_params):
 
         self.build_fn = build_fn
 
-        if sk_params:
+        kwargs = (
+            _get_default_args(build_fn) if inspect.isfunction(build_fn) else {}
+        )
+        sk_params = {**kwargs, **sk_params}
```

Comment:
I think this is a bit easier to understand:

```suggestion
       if inspect.isfunction(build_fn):
            kwargs = _get_default_args(build_fn)
            sk_params = {**kwargs, **sk_params}
```